docs/statsig-warehouse-native/features/autotune.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Autotune
+slug: /statsig-warehouse-native/features/autotune
+sidebar_label: Autotune
+---
+
+Autotune Experiments in Warehouse Native have a very similar setup to cloud [Autotunes](https://docs.statsig.com/autotune). The key difference is source of the success event that the Autotune will optimize for.
```

Comment:
In Warehouse Native, Autotune success events will be pulled from a metric source, and computation of results will happen in your warehouse.